examples/step-68/step-68.cc

```diff
@@ -0,0 +1,810 @@
+
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_system.h>
+#include <deal.II/fe/mapping_q.h>
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/grid_tools.h>
+#include <deal.II/grid/tria_accessor.h>
+#include <deal.II/grid/tria_iterator.h>
+
+#include <deal.II/lac/la_parallel_vector.h>
+#include <deal.II/lac/vector.h>
+
+#include <deal.II/numerics/data_out.h>
+#include <deal.II/numerics/vector_tools.h>
+
+// From the following include file we import the ParticleHandler class
+// that allows you to manage
+// a collection of particles (objects of type Particles::Particle), representing
+// a collection of points with some attached properties (e.g., an id) floating
+// on a parallel::distributed::Triangulation. The methods and classes in the
+// namespace Particles allows one to easily implement Particle-In-Cell methods
+// and particle tracing on distributed triangulations:
+#include <deal.II/particles/particle_handler.h>
+
+// We import the particles generator
+// which allow us to insert the particles. In the present step, the particle
+// are globally inserted using a non-matching hyper-shell triangulation:
+#include <deal.II/particles/generators.h>
+
+// Since the particles do not form a triangulation, they have their
+// own specific DataOut class which will enable us to write them
+// to commonly used parallel vtu format:
+#include <deal.II/particles/data_out.h>
+
+#include <cmath>
+#include <iostream>
+
+namespace Step68
+{
+  using namespace dealii;
+
+  // @sect3{Run-time parameter handling}
+
+  // Similarly to what is done in step-60, we set up a class that holds
+  // all the parameters of our problem and derive it from the ParameterAcceptor
+  // class to simplify the management and creation of parameter files.
+  //
+  // The ParameterAcceptor paradigm requires all parameters to be writable by
+  // the ParameterAcceptor methods. In order to avoid bugs that would be very
+  // difficult to track down (such as writing things like `time = 0` instead of
+  // `time == 0`), we declare all the parameters in an external class, which is
+  // initialized before the actual `ParticleTracking` class, and pass it to
+  // the main class as a `const` reference.
+  //
+  // The constructor of the class is responsible for the connection between the
+  // members of this class and the corresponding entries in the
+  // ParameterHandler. Thanks to the use of the
+  // ParameterHandler::add_parameter() method, this connection is trivial, but
+  // requires all members of this class to be writable.
+  class ParticleTrackingParameters : public ParameterAcceptor
+  {
+  public:
+    ParticleTrackingParameters();
+
+    // This class consists largely of member variables that
+    // describe the details of the particle tracking simulation and its
+    // discretization. The following parameters are about where output should
+    // land, the spatial discretization of the velocity (the default is $Q_1$),
```

Comment:
Following @peterrum and @gassmoeller 's suggestion
```suggestion
    // written to, the spatial discretization of the velocity (the default is $Q_1$),
```